python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -443,6 +452,26 @@ def open(self, params: PipesParams) -> Iterator[T_MessageChannel]:
             PipesMessageWriterChannel: Channel for writing messagse back to Dagster.
         """
 
+    @final
```

Comment:
ha first time i've seen this in the wild

Reply:
Yeah I didn't know it existed until now